warehouse/forklift/legacy.py

```diff
@@ -46,6 +50,17 @@
 
 PATH_HASHER = "blake2_256"
 
+ALL_STDLIB = set(
+    chain.from_iterable(
+        [stdlib_list.stdlib_list(version)
+         for version in stdlib_list.short_versions]
+    )
+)
+
+STDLIB_PROHIBITTED = [
```

Comment:
Small nitpick, but this would probably be better as a set, it will save some duplication for like ``ConfigParser`` vs ``configparser`` and will also make the ``in`` check later on a tiny bit faster.